2996_reflection/reflection.md

```diff
@@ -7972,17 +7989,18 @@ Produces an injected declaration `$D$` ([expr.const]) that provides a definition
 
 ::: std
 ::: addu
-[1]{.pnum} Subclause [meta.reflection.unary] contains consteval functions to query the properties of a type at compile time.
+[1]{.pnum} Subclause [meta.reflection.unary] specifies consteval functions to query the properties of a type during translation.
```

Comment:
```suggestion
[1]{.pnum} Subclause [meta.reflection.unary] specifies consteval functions to query the properties of a type at compile time.
```
This is Library wording, Library talks about compile time, we're not changing this one.